app/components/earthworks/also_available_component.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Earthworks
+  # Display also available as links in sidebar
+  class AlsoAvailableComponent < ViewComponent::Base
+    attr_reader :document
+
+    def initialize(document:, **)
```

Comment:
do we need the **?

Reply:
nope, not sure why i put that there (probably copy-pasta)